src/libs/mysqlHandler.go

```diff
@@ -18,6 +18,7 @@ const TableNetworkPolicy_TableName = "network_policy"
 const TableSystemPolicy_TableName = "system_policy"
 const TableSystemLogs_TableName = "system_logs"
 const TableNetworkLogs_TableName = "network_logs"
+const PolicyTable_TableName = "policy_table"
```

Comment:
A table with name `policy_"table"`? :slightly_smiling_face:

Reply:
I know. I cannot come up with any other name. We can say maybe policy_yaml_table.

Reply:
I can understand the struggle :slightly_smiling_face: | `policy_yaml` should be fine. Let's not add `table` as the suffix.